src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -1271,10 +1265,10 @@ subroutine Initialize(gc, import, export, clock, rc)
        nhms = h *10000 + m*100  + s
 
        do ens = 0, NUM_ENSEMBLE-1
-          call get_id_string(id_string, ens + FIRST_ENS_ID, ens_id_width)
+          call get_ensid_string(ensid_string, ens + FIRST_ENS_ID, ens_id_width, NUM_ENSEMBLE ) ! xxxx not _exxxx
```

Comment:
I don't understand the comment here:   `...  ! xxxx not _exxxx`
Doesn't get_ensid_string produce "_eXXXX"?
[Same comment applies to line ~1591 below.]

Reply:
yes. For backward compatibility, I didn't change the file name template, so it should be right here.